src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1565,6 +1566,24 @@ public String toVariableNameString(EnchantmentOffer eo) {
 					"Individual attributes of a snapshot cannot be modified or retrieved.")
 				.requiredPlugins("Minecraft 1.20.2+")
 				.since("INSERT VERSION")
+				.parser(new Parser<>() {
+					@Override
+					public boolean canParse(ParseContext context) {
+						return false;
+					}
+
+					@Override
+					public String toString(EntitySnapshot snapshot, int flags) {
+						if (supportsGetAs)
+							return snapshot.getAsString();
```

Comment:
according to the javadocs:
"Get this EntitySnapshot as an NBT string."

Is this really what Skript wants when stringifying something?

Reply:
It was a brief discussion between burb Pikachu. 
Pikachu said "I think it could be reasonable to have the toString be getAsString"
Burb sends a ss of the `getAsString`
Pikachu "You asked it to send it to you, and now your mad it sent it to u?"

Not really a definite answer, but I figured why not.

Reply:
In my opinion it doesn't flow with Skript.
It would be the equivalent of sending the full NBT of an ItemStack/ItemType when stringifying.
You know me, I do love my NBT 😂 ... but in this case I just think its a bit odd.